src/entrypoint/nsinstance.go

```diff
@@ -353,22 +326,15 @@ func (n *NSInstance) sendTerminate(force bool) {
 	}
 
 	var (
-		launcher = n.launcherCmd != nil && n.launcherCmd.ProcessState != nil && !n.launcherCmd.ProcessState.Exited()
-		waiter   = n.waiterCmd != nil && n.waiterCmd.ProcessState != nil && !n.waiterCmd.ProcessState.Exited()
+		game = n.gameCmd != nil && n.gameCmd.ProcessState != nil && !n.gameCmd.ProcessState.Exited()
 	)
 
-	if launcher {
-		if force {
-			n.launcherCmd.Process.Signal(syscall.SIGKILL)
-		} else {
-			n.launcherCmd.Process.Signal(syscall.SIGTERM)
-		}
-	}
-
-	if launcher || waiter {
+	if game {
 		if force {
+			n.gameCmd.Process.Signal(syscall.SIGKILL)
 			n.winecmd("wineserver", "--kill").Start()
 		} else {
+			n.gameCmd.Process.Signal(syscall.SIGTERM)
```

Comment:
This one isn't necessary; wineboot will handle it and send the proper window close messages.

```suggestion
```